google/cloud/aiplatform/jobs.py

```diff
@@ -533,7 +542,8 @@ def _create(
         cls,
         api_client: job_service_client.JobServiceClient,
         parent: str,
-        batch_prediction_job: gca_bp_job.BatchPredictionJob,
+        batch_prediction_job: gca_bp_job_default.BatchPredictionJob,
```

Comment:
This seems like the type should be a Union of v1 and v1beta1 BatchPredictionJob.